src/messages.pot

```diff
@@ -17,11 +17,15 @@ msgstr ""
 "Content-Type: text/plain; charset=CHARSET\n"
```

Comment:
As stated, as far as I'm seeing, this file is not used.

Reply:
I agree. Would you mind deleting it as part of this PR?

Reply:
Done!